data/notices.json

```diff
@@ -779,6 +779,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(cli): CLI versions and CDK library versions have diverged",
+      "issueNumber": 32775,
+      "overview": "Starting in CDK 2.179.0, CLI versions will no longer be in lockstep. CLI versions will now be released as 2.1000.0 and continue with 2.1001.0, etc.",
+      "components": [
+        {
+          "name": "framework",
+          "version": ">=2.179.0"
```

Comment:
im intending to target 2.179.0 and every version after to force customers to acknowledge this notice regardless of what version they end up upgrading to. is this the right thing to do?

Reply:
I would trigger on the CLI version instead of the framework version? And probably only for the first few? `>= 2.1000.0 < 2.1005.0` or sth

Reply:
I would target _all_ old versions to let users know that the version jump is expected. One they've upgraded, it's already to late.

Reply:
How about we combine both @rix0rrr and @mrgrain suggestions? Triggering on the CLI, but only with an upper bound: `< 2.1005.0`. Before the upgrade it serves as an advanced notice. After the upgrade, it serves to preempt the question "WTF? Why are my versions different now?"